conda_forge_tick/auto_tick.py

```diff
@@ -394,6 +396,11 @@ def add_rebuild_migration_yaml(
     # post plucking we can have several strange cases, lets remove all selfloops
     total_graph.remove_edges_from(nx.selfloop_edges(total_graph))
 
+    package_names = {
+        p if p in gx.nodes else output_to_recipe[p]
+        for p in package_names
+    }
```

Comment:
What happened to exclude packages? That was important for the python migration?

Reply:
It got dropped as I edited a outdated branch when debugging why #853 didn't work properly and the rebased badly.

The exclusions are still applied on line 459 so it still behaves in almost the same way. The only change is that if `exclude_packages` contains a package with multiple `outputs`, the output name will also have to be added to `exclude_packages`. For example if `matplotlib` and `matplotlib_base` both depend on `python` both would have to be excluded explicitly.

Thinking about it more, I guess this is probably just annoyingly verbose and `exclude_packages` is actually `exclude_feedstocks`? If you agree I'll make a PR.

Reply:
That's an interesting point. I would go with `exclude_feedstocks`. Its usage is rare (eg to remove compilers from the python graph) so I think we can make the assumption that you want to remove the entire feedstock, since we don't have the capacity to issue PRs on individual outputs.

Reply:
Done in https://github.com/regro/cf-scripts/pull/855 😃